breakable_toy/scripts/db-connect

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker-compose exec -u postgres database psql -d postgres
```

Comment:
:+1: Good impulse to encapsulate the common but non-obvious tasks.  I meant to send you https://github.com/azavea/architecture/blob/master/doc/arch/adr-0000-scripts-to-rule-them-all.md, but I don't think you have access to it.  I asked Ops to add it.  Since projects often end up with quite a lot of services, our pattern is to make one script (which we call `console`, though that seems more apt for some services than others) that takes the name of the particular service as an argument.